package.json

```diff
@@ -84,6 +84,7 @@
     "karma-sinon-chai": "1.2.0",
     "lazypipe": "1.0.1",
     "lolex": "1.4.0",
+    "markdown-link-check": "3.0.3",
```

Comment:
the `yarn.lock` file needs updating. if you use `yarn add markdown-link-check` it would add it to both files.